### PR TITLE
Issue 10

### PR DESCRIPTION
**Pull request by jessesquires**

### CODE REVIEW

**Comment by dellsystem** [comment]
I used a wikicommons version of the photo that Vikram suggested (not sure about usage rights for the original photo) and a slightly shorter version of his original caption. The content could use another proofread, especially for formatting errors (Dropbox's exported markdown is often buggy). Here's a screenshot:

image

**Review by dellsystem — APPROVED** [review]

File: _posts/2020-09-04-issue-10.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+[Duane Deterville](https://africana.sfsu.edu/people/faculty/duane-deterville), a visual culture scholar in the Africana Studies Department at San Francisco State University who teaches “[Black On-Line: Cyberspace, Culture, and Community](http://userwww.sfsu.edu/def/cyb.html),” highlights just how important these other storylines were. He points us to [one Hudlin comic](http://hudlinentertainment.com/black-panther-annual-1-review/) where T’Challa, a grey-haired elder and world leader, tells his granddaughter the cautionary tale of American empire:
+> "...we thought that level of ambition would damage the morality of our culture. Imperialism becomes a self-serving goal that undermines what you're fighting for. Look what happened to Christianity once they went from being an oppressed religion to persecuting 'non-believers' themselves."
+
+![A page from Reggie Hudlin’s Black Panther, all rights reserved. Source: hudlinentertainment.com](https://paper-attachments.dropbox.com/s_88265DB218C0973A3D0FB0A4EB1F3785283561E097A84C2026BCFA0022EE533F_1599174301784_image.png)
```
Comment (jessesquires) [inline _posts/2020-09-04-issue-10.md:89]: yeah we can't hotlink this from dropbox paper

**Comment by jessesquires** [comment]
i'm fixing images and various markdown bugs now. will push updates soon.

File: _posts/2020-09-04-issue-10.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ At almost 40, I realize that majority of us tech workers look nothing like what
...
 
-I don’t think it’s asking too much that I earn a decent salary, have job stability, keep reasonable hours, have autonomy in my work, and feel that I can shape the direction and values of my company. Startup employees are often lured in with the promise that they’ll own stock in the company, but owning 0.001% of a company when the CEO owns 30% is a far cry from co-ownership. In practice, startups are rarely the egalitarian dream their founders present them to be. Many tech workers, myself included, would rather be baking bread.
+I don’t think it’s asking too much that I earn a decent salary, have job stability, keep reasonable hours, have autonomy in my work, and feel that I can shape the direction and values of my company. Startup employees are often lured in with the promise that they’ll own stock in the company, but owning 0.05% of a company when the CEO owns 30% is a far cry from co-ownership. In practice, startups are rarely the egalitarian dream their founders present them to be. Many tech workers, myself included, would rather be baking bread.
```
Comment (jessesquires) [inline _posts/2020-09-04-issue-10.md:37]: i don't think this is accurate. people really do get `.001` percent of a company. that wasn't an exaggeration.

**Comment by jessesquires** [comment]
ok i think we're good to go

**Pull request merged**